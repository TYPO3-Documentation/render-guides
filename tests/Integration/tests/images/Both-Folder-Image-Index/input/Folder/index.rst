==============
Document Title
==============

Lorem Ipsum Dolor.

.. figure:: /Images/typo3-logo.png
   :class: with-shadow
   :alt: Content element wizard with the new content element

    Content element wizard with the new content element
