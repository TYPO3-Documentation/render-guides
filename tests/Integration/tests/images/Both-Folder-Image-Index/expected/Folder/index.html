<!-- content start -->
                <section class="section" id="document-title">
            <h1>Document Title<a class="headerlink" href="#document-title" data-bs-toggle="modal" data-bs-target="#linkReferenceModal"  title="Reference this headline">Â¶</a></h1>

    <p>Lorem Ipsum Dolor.</p>

            <figure class="with-shadow">
    <img
    src="../Images/typo3-logo.png"
                alt="Content element wizard with the new content element"        />



                    <figcaption>
    <p>Content element wizard with the new content element</p>
</figcaption>
            </figure>

    </section>
        <!-- content end -->
